spec/src/features/tree-spec.coffee

```diff
@@ -55,7 +55,7 @@ describe '"tree" control', ->
     template.form(dom, model: model, attributes: attrs)
     expect($('.jstree-clicked').parent().attr('node_value')).toBe("/GLAH01/Data_1HZ_VAL/Engineering/d_T_detID_VAL");
 
-  it "will not allow irrelevant nodes to be selected", ->
+  it "will render irrelevant nodes properly and not allow them to be selected", ->
```

Comment:
it "renders..." rather than it "will render". Consider splitting into separate specs.